repose-aggregator/components/filters/cors-filter/src/main/scala/org/openrepose/filters/cors/CorsFilter.scala

```diff
@@ -44,18 +48,18 @@ class CorsFilter @Inject()(configurationService: ConfigurationService)
 
   import CorsFilter._
 
-  private var configurationFile: String = DEFAULT_CONFIG
+  private var configurationFile: String = _
   private var initialized = false
-  private var allowedOrigins: Iterable[Regex] = _
-  private var allowedMethods: Iterable[String] = _
-  private var resources: Iterable[Resource] = _
+  private var allowedOrigins: Seq[Regex] = _
+  private var allowedMethods: Seq[String] = _
+  private var resources: Seq[Resource] = _
```

Comment:
Out of a curiosity, was there a particular reason for this change? I'd like to thing something came out of our discussion on the subject.